boards/arm/nrf5340dk_nrf5340/nrf5340_cpuapp_common-pinctrl.dtsi

```diff
@@ -8,6 +8,7 @@
 		group1 {
 			psels = <NRF_PSEL(TWIM_SDA, 1, 2)>,
 				<NRF_PSEL(TWIM_SCL, 1, 3)>;
+			bias-pull-up;
```

Comment:
This should not be added at the board level. If a shield needs to use SoC internal pull-ups, then it should add them in its overlay. Normally, external pull-ups should be used for I2C bus.

Reply:
Boards like nrf52840dk have external pull up resistors, I did not see any pull up resistors on the nrf5340dk, hence why I added them here

Reply:
Those are present also on the nRF5340 DK, see [Signal switches](https://infocenter.nordicsemi.com/topic/ug_nrf5340_dk/UG/dk/signal_switches.html) (U6 specifically).

Reply:
Not sure how I missed that, right you are.